.circleci/config.yml

```diff
@@ -145,13 +145,13 @@ workflows:
     jobs:
       - valgrind-test
 
-  # Since valgrind tests really take a long time to finish, wait for 9.5 hours.
+  # Since valgrind tests really take a long time to finish, wait for 10.5 hours.
   # Then push valgrind test results and terminate the machine.
   weekly-valgrind-finalize:
     triggers:
       - schedule:
-          # https://crontab.guru/#30_9_*_*_1
-          cron: "30 9 * * 1"
```

Comment:
Should we increase this more since we are adding more tests I guess we will hit this limit soon again?

Reply:
makes sense to me, what about 2.5 hours ?

Reply:
Sure, another question I have is do we have an easy way of understanding if the time is not enough again? Did you manually understand that?

Reply:
yes, I did that manually, by looking into the result that weekly job pushed:

https://github.com/citusdata/release-test-results/blob/citusbot_valgrind_test_resource_group/05_24_2021_1621848783/periodic_job_results/05_24_2021_1621848783/valgrind_test_out.txt